ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/util/BeaconStateUtil.java

```diff
@@ -442,7 +495,9 @@ public static int get_epoch_committee_count(int active_validator_count) {
     List<Integer> active_validator_indices =
         ValidatorsUtil.get_active_validator_indices(validators, UnsignedLong.valueOf(epoch));
 
-    int committees_per_epoch = get_epoch_committee_count(active_validator_indices.size());
+    // TODO: revisit when we figure out what to do about integer indexes
```

Comment:
Can you confirm an issue is created for this too. Or just add this as a comment to #303. This definitely has loss of precision bug written all over it.